docs/src/tutorial.md

```diff
@@ -0,0 +1,139 @@
+# Tutorial
+
+HSL MA57 and MA97 can be used for the solution of symmetric, possibly indefinite, linear systems.
+They are often used for the solution of saddle-point systems, i.e., systems of the form
+```math
+\begin{bmatrix}
+H & A^T \\
+A &
+\end{bmatrix}
+\begin{bmatrix}
+x \\ y
+\end{bmatrix}
+=
+\begin{bmatrix}
+b \\ c
+\end{bmatrix}.
+```
+Special cases occur when ``H`` is the identity matrix and either ``b = 0`` or ``c = 0``, which correspond to the least-squares problem
+```math
+\min_y \ \|A^T y - b\|
+```
+and to the least-norm problem
+```math
+\min_x \ \|x\| \ \text{subject to } Ax = c
+```
+respectively.
+
+## HSL_MA97
+
+Supported versions:
+* 2.6.0
+* 2.7.0
+
+[HSL_MA97](http://www.hsl.rl.ac.uk/catalogue/hsl_ma97.html): an OpenMP-based direct solver for symmetric linear systems. Example:
+
+```JULIA
+using MatrixMarket
+using HSL
+
+K = MatrixMarket.mmread("K.mtx")  # only the lower triangle
+rhs = readdlm("rhs.rhs")
+
+LBL = Ma97(K)
+ma97_factorize!(LBL)
+x = ma97_solve(LBL, rhs)  # or x = LBL \ rhs
+```
+
+### Rectangular Systems
+
+There is a convenience interface to solve rectangular systems that complements
+the sparse QR factorization in Julia.
+
+When *A* is *m*-by-*n* with *m* < *n* and has full row rank,
+```JULIA
+(x, y) = ma97_solve(A, b)
+```
+solves for the minimum-norm solution, i.e., *x* such that *Ax = b* and *x +
+Aᵀ y = 0*. The call
```

Comment:
Is it possible to refer to the min-norm problem defined above? (using c instead of b.)

Reply:
Done, thanks!

Reply:
I was wondering if it's possible to refer to an equation number.

Reply:
Done. https://juliasmoothoptimizers.github.io/HSL.jl/previews/PR93/tutorial/